indico/modules/categories/templates/management/category_protection.html

```diff
@@ -15,7 +15,7 @@
     {% endcall %}
     {% call form_footer(form) %}
         <input class="i-button big highlight" type="submit"
-               value="{% trans %}Save{% endtrans %}" data-disabled-until-change>
+               value="{% trans %}Save{% endtrans %}" data-disabled-until-change data-save-reminder>
```

Comment:
As mentioned a few more times, it's visually nicer when we break the attributes into a new line each:
```html
<input class="i-button big highlight"
       type="submit"
       value="{% trans %}Save{% endtrans %}"
       data-disabled-until-change
       data-save-reminder>
```
However, it's still a matter of preference so no strong opinion!